ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -21,6 +21,25 @@ public final class AliasTokens {
         case brandPressed
         case brandSelected
         case brandDisabled
+        // New foreground colors
```

Comment:
Will the old colors be removed/replaced eventually?

Reply:
Yep! We're keeping them in to avoid any issues with controls currently using them. But unused colors will be progressively removed.

Reply:
Yes they will, but not until we've removed all references to them.